From 50ac003e455a53a5f10c382ae8bb345e348e24ea Mon Sep 17 00:00:00 2001
From: Wenping Zhang <wenping.zhang@rock-chips.com>
Date: Fri, 30 Jun 2017 17:20:18 +0800
Subject: [PATCH] ARM: dts: rk3229-gva-sdk: delete vcc_host node in gva dts.

Gva products don't use usb host mode, so delete vcc_host regulator
node and other related configuration.

Change-Id: I9d8db775ab33ab7b51c2a4d867d77e1fe8912d9a
Signed-off-by: Wenping Zhang <wenping.zhang@rock-chips.com>
---
 arch/arm/boot/dts/rk3229-gva-sdk.dts | 19 -------------------
 1 file changed, 19 deletions(-)

diff --git a/arch/arm/boot/dts/rk3229-gva-sdk.dts b/arch/arm/boot/dts/rk3229-gva-sdk.dts
index e6ff5764eb37..c976829200ee 100644
--- a/arch/arm/boot/dts/rk3229-gva-sdk.dts
+++ b/arch/arm/boot/dts/rk3229-gva-sdk.dts
@@ -66,17 +66,6 @@
 		reset-gpios = <&gpio2 26 GPIO_ACTIVE_LOW>; /* GPIO2_D2 */
 	};
 
-	vcc_host: vcc-host-regulator {
-		compatible = "regulator-fixed";
-		enable-active-high;
-		gpio = <&gpio3 20 GPIO_ACTIVE_HIGH>;
-		pinctrl-names = "default";
-		pinctrl-0 = <&host_vbus_drv>;
-		regulator-name = "vcc_host";
-		regulator-always-on;
-		regulator-boot-on;
-	};
-
 	wireless-bluetooth {
 		compatible = "bluetooth-platdata";
 		clocks = <&rk805 1>;
@@ -286,11 +275,6 @@
 		};
 	};
 
-	usb {
-		host_vbus_drv: host-vbus-drv {
-			rockchip,pins = <3 20 RK_FUNC_GPIO &pcfg_pull_none>;
-		};
-	};
 };
 
 &sdio {
@@ -304,6 +288,3 @@
 	status = "disabled";
 };
 
-&u2phy0_host {
-	phy-supply = <&vcc_host>;
-};
-- 
2.35.3

